src/domain/interfaces/transaction-api.interface.ts

```diff
@@ -27,4 +27,13 @@ export interface ITransactionApi {
   getMasterCopies(): Promise<MasterCopy[]>;
 
   getSafe(safeAddress: string): Promise<Safe>;
+
+  getDelegates(
```

Comment:
Just as a comment, not required for this PR, but we could start thinking in wrapping arguments into DTO types when they're a bunch as in this case.